docs: preview not for production on CLI guide (#13316)
vitejs/vite: docs/guide/cli.md

### `vite preview`

> Locally preview production build. This command is only for previewing the build locally and not designed as a production server.

Reviewer: The second sentence duplicates the first one a bit. Maybe 🤔 
```suggestion
Locally preview the production build. Do not use this as a production server as it's not designed for it.
```
